src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+@use "../utils/variables" as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  border: 1px solid $gray-element;
+  border-radius: 5px;
+  background-image: url("../images/imac.jpeg");
```

Comment:
please use tag img insted and add also alt to it, it will make screen readers know that you have some image of your product :wink: